src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -4009,6 +4009,7 @@ public Response addGlobusFilesToDataset(@Context ContainerRequestContext crc,
         logger.info(" ====  (api addGlobusFilesToDataset) jsonData   ====== " + jsonData);
 
         if (!systemConfig.isHTTPUpload()) {
+            // @todo why isHTTPUpload()? - shouldn't it be checking isGlobusUpload() here? 
```

Comment:
Ok, fixed. Maybe it wasn't really necessary, I guess, since the API methods that need to be called first had proper globus availability checks. But then perhaps there is a use case where an admin would want to disable http and only have globus uploads?